shopelectro/selenium/elements/product_card.py

```diff
@@ -0,0 +1,19 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class ProductCard:
+    """"Represent a product card of category page."""
+
+    def __init__(self, driver, card_number):
```

Comment:
it'll be good to provide doc for card_number: `Order number at category page's card list`